src/main/java/org/jabref/model/entry/Month.java

```diff
@@ -85,9 +89,13 @@
             testString = testString.substring(0, 3);
         }
         Optional<Month> month = Month.getMonthByShortName(testString);
+        Optional<Month> monthGerman = Month.parseGermanShortMonth(testString);
```

Comment:
Please move that after line 95. the German month only has to be determined if there is no English month.

Reply:
Done.